### ISSUE TITLE
New license request: FreeBSD-DOC [SPDX-Online-Tools]

### ISSUE THREAD

**Issue by shujisado** [spdx/license-list-XML#1111]
**1.** License Name: FreeBSD Documentation License
**2.** Short identifier: FreeBSD-DOC
**3.** License Author or steward: The FreeBSD Project
**4.** Comments: This license is the license in common use for FreeBSD-related documentation.
**5.** Standard License Header: 
**6.** URL: https://www.freebsd.org/copyright/freebsd-doc-license.html
**7.** OSI Status: Not Submitted

**Comment by johnmhoran** [comment]
@swinslow I'll tackle this submission.

**Comment by johnmhoran** [comment]
TL;DR: Given the stature of The FreeBSD Project and the popularity of its BSD distributions, I think this license should be added.

Notes:

It does not appear that the submitter (https://github.com/shujisado) has any direct connection to the asserted author/steward (The FreeBSD Project: https://www.freebsd.org/). 

The proposed license appears to be based on "The FreeBSD Copyright" (https://www.freebsd.org/copyright/freebsd-license.html), with changes made to refer to documentation rather than source code, as well as the addition of a section titled "Manual Pages" that refers to "pages [that] contain text from the IEEE Std 1003.1, 2004 Edition, Standard for Information Technology -- Portable Operating System Interface (POSIX®) specification."

Note that "The FreeBSD Copyright" license is included on the SPDX license list as "BSD 2-Clause FreeBSD License" (short identifier: "BSD-2-Clause-FreeBSD") and has been deprecated.  See https://spdx.org/licenses/BSD-2-Clause-FreeBSD.html. 

Third-party references to the proposed license include:

Microsoft Teams Third Party Notice for iOS
https://support.microsoft.com/en-us/office/microsoft-teams-third-party-notice-for-ios-2cb2ca96-40d1-4282-aabe-5c5c4bcf35c7 
- Includes license text ("Copyright 1994-2016 The FreeBSD Project.").
- Most recent date for copyright notices on that page (other than the Microsoft notice at the bottom, which is 2021) is 2017.

Cisco NAC Profiler Installation and Configuration Guide, Release 3.1
https://content.cisco.com/chapter.sjs?uri=/searchable/chapter/www.cisco.com/content/en/us/td/docs/security/nac/profiler/configuration_guide/310/nac_profiler_book/p_apx_open31.html.xml 
- Includes license text ("Copyright 1994-2009 The FreeBSD Project.").
- "Published On: August 5ᵗʰ, 2019 19:05"

gnu.org -- listed under "Licenses For Documentation" as "FreeBSD Documentation License"
https://www.gnu.org/licenses/license-list.en.html#FreeBSDDL 
- Described as "a permissive non-copyleft free documentation license that is compatible with the GNU FDL", though note that the link leads to a page containing the text of the source code license, not the documentation license (https://directory.fsf.org/wiki/License:BSD-2-Clause-FreeBSD). 

Some third parties have adapted the text of the proposed license for their own purposes.  See, e.g.,

Installing and Using Common Lisp Libraries
http://rpgoldman.real-time.com/lisp/fight-the-system.html 

- includes similar but not identical version of the 1st of the 2 parts of the submitted license

OcempGUI Manual
http://ocemp.sourceforge.net/manual/ocempgui-manual.html 

- includes similar but not identical version of the 1st of the 2 parts of the submitted license

**Comment by swinslow** [comment]
+1 -- thanks for the writeup, @johnmhoran, I agree this should be added.

**Comment by emmanueltournier** [comment]
+1

**Comment by jlovejoy** [comment]
+1 to add from me too

**Comment by jlovejoy** [comment]
fixed by #1175
